.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+<!--
+Thanks for the Pull Request!
+
+Please review [Contribution Guide](https://github.com/CesiumGS/cesium/blob/main/CONTRIBUTING.md) before opening your first Pull Request.
+
+To ensure your Pull Request is reviewed and accepted quickly, please refer to our [Pull Request Guidelines](https://github.com/CesiumGS/cesium/blob/main/CONTRIBUTING.md#pull-request-guidelines).
+
+--->
+
+# Description
+
+<!--- Describe your changes in detail -->
+
+<!--- Consider: Why is this change required? What problem does it solve? -->
+
+<!--- Include screenshots if appropriate --->
```

Comment:
I think it would be better to condense these to a single comment that pr authors can write in before or after. Unless the intention is that authors specifically answer each of these separately then it works as is.

```suggestion
<!--
  Describe your changes in detail
  Consider: Why is this change required? What problem does it solve?
  Include screenshots if appropriate
-->
```

Reply:
Though I will fix the inconsistent number of `-` character on the single comments if you agree with keeping them separate.

Reply:
I think that it's likely there would be cases of myself or others filling out the template and doing parts 2 and 3 under the first then having to extract that information to the other parts if that's the intention. I was thinking along the lines of it being better to include the whole "prompt" and let the author decide how to structure it all together.

I don't have a strong opinion either way though. If you wanna stick with separate comments that's fine, just update the `-` and I can merge